src/config/playerWeapons.js

```diff
@@ -11,18 +11,67 @@ export const PRIVATE_SLACK_CHANNEL: Weapon = {
   name: 'Private Slack Channel',
   image: slackImg,
   attacks: [
-    { type: 'exclusionary', name: 'Master Lock Combo' },
-    { type: 'exclusionary', name: 'Double Padlock' },
-    { type: 'exclusionary', name: 'Insult Sling' },
-    { type: 'exclusionary', name: '“Cool People”' },
-    { type: 'exclusionary', name: '“Real Artists”' },
-    { type: 'exclusionary', name: 'Talk to the Hand' },
-    { type: 'exclusionary', name: 'Stranger Danger' },
+    {
+      type: 'exclusionary',
+      name: 'Master Lock Combo',
+      description:
+        '[Name] goes at [BOSS] with their infamous attack: The Master Lock Combination Spin',
```

Comment:
having `[Name]` be in sentence case and `[BOSS]` be in all caps is stressing me out but I guess we can fix that once it's all loaded in lol